src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/BundleRename.cs

```diff
@@ -22,7 +22,7 @@ public BundleRename(SharedTestState fixture)
             sharedTestState = fixture;
         }
 
-        [Theory]
+        [Theory(Skip = "Fails in CI: https://github.com/dotnet/runtime/issues/35068")]
```

Comment:
We use [ActiveIssue(...)] for such things in the library tests.  I've not paid attention in the past to what's used in the installer tests.  Are we able to use [ActiveIssue(...)] in such cases as well?  If we haven't before, don't worry about it for this PR; in the future, it'd be nice to align across all the tests in the repo.

Reply:
ActiveIssue should work fine as well in installer.